node/node_bridge_test.go

```diff
@@ -8,30 +8,10 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-
-	"github.com/libp2p/go-libp2p-core/host"
-	"github.com/libp2p/go-libp2p-core/network"
-	"github.com/libp2p/go-libp2p-core/protocol"
 )
 
-func TestNewBridge(t *testing.T) {
-	store := MockStore(t, DefaultConfig(Bridge))
-	node, err := New(Bridge, store)
-	require.NoError(t, err)
-	require.NotNil(t, node)
-	require.NotNil(t, node.Config)
-	require.NotNil(t, node.Host)
-	require.NotNil(t, node.HeaderServ)
-	require.Nil(t, node.BlockServ)
-	assert.NotZero(t, node.Type)
-}
-
-func TestBridgeLifecycle(t *testing.T) {
-	cfg := DefaultConfig(Bridge)
-	store := MockStore(t, cfg)
-
-	node, err := New(Bridge, store)
-	require.NoError(t, err)
+func TestNewBridgeAndLifecycle(t *testing.T) {
```

Comment:
Not sure why we separated those tests initially as they can be simplified into one.